### DISCUSSION TITLE
New setup

### DISCUSSION THREAD

**Discussion by daf0555** [homebridge-simplisafe3/homebridge-simplisafe3#257] [Q&A] [answered]
Sorry if this is basic. I am trying to set up Simplisafe on Homebridge VM so it can show up on my Homekit. I have the plugin installed. When I click the button, it goes to the OAuth page to login. Once logged in, I get to the home dashboard of SimpliSafe. Being in Chrome, I go to developer tools to pull up the error message to copy the link. However, I do not see any links that start with (com.simplisafe.moble://). Thanks for any help.

**Comment by shamoon** [comment, marked as answer]
Hi @daf0555 , it shouldn’t take you to the SimpliSafe dashboard. After you click to verify your email the original page in Chrome should try to redirect but fail (so you see a blank page) and that’s where you look in the error console. Not sure exactly where things are going off track for you.

**Reply by daf0555** [reply]
Thanks. That did it! I was just going too far.
